src/mpol/fourier.py

```diff
@@ -19,10 +19,11 @@ class FourierCube(nn.Module):
         cell_size (float): the width of an image-plane pixel [arcseconds]
         npix (int): the number of pixels per image side
         coords (GridCoords): an object already instantiated from the GridCoords class. If providing this, cannot provide ``cell_size`` or ``npix``.
+        device (torch.device) : the desired device of the image cube. If ``None``, defalts to current device.
 
     """
 
-    def __init__(self, cell_size=None, npix=None, coords=None):
+    def __init__(self, cell_size=None, npix=None, coords=None, device=None):
```

Comment:
Following @kadri-nizam 's discussion, I don't think it's necessary to specify `device` in this way. So long as the relevant aspects of the layer are properly registered as buffers or parameters, then if we had something like 

`fcube = Fourier(...)`

we could just do `fcube.to('cuda')`, which seems to follow PyTorch idioms.

Reply:
I think similar comments apply to gridding and precomposed.

Reply:
Device calls have now been removed.